js/views/RegistrationPagePart3.js

```diff
@@ -61,11 +61,11 @@ const RegistrationPagePart3 = React.createClass ({
           button_text: 'Finish'
         });
       });
-      return;
```

Comment:
There is no effective change here.

http://programmers.stackexchange.com/questions/18454/should-i-return-from-a-function-early-or-use-an-if-statement